TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Common/UiDefaults.cs

```diff
@@ -0,0 +1,7 @@
+namespace TeachingRecordSystem.SupportUi.Pages.Common;
+
+public static class UiDefaults
+{
+    public const string DefaultNullDisplayContent = "Not provided";
+    public const string DefaultDateOnlyDisplayFormat = "d MMMM yyyy";
+}
```

Comment:
Don't think we need `Default` in each member's name if it's in the class name already.

```suggestion
public static class UiDefaults
{
    public const string EmptyDisplayContent = "Not provided";
    public const string DateOnlyDisplayFormat = "d MMMM yyyy";
}
```